nebula-components/objects/_ui-list.scss

```diff
@@ -0,0 +1,15 @@
+@if $nb-use-ui-list {
+  .#{$nb-namespace}o-ui-list {
+    @include nb-ui-list;
```

Comment:
This should just be `@include nb-bare-list`

Reply:
done